src/stories/cartesian/brush/BrushComponent.jsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable no-useless-constructor */
+/* eslint-disable import/no-default-export */
+import React, { Component } from 'react';
+import { Surface, Brush } from '../../..';
```

Comment:
Not sure if it's better to replace the import with an explicit path, just an idea. I faced a problem in my project with import path end with `'..'`. So to avoid potential problem, please consider the suggestion.
```typescript
import { Surface } from '../../../container/Surface';
import { Brush } from '../../../cartesian/Brush';
```

Reply:
I thought the path ending in .. was odd, too. But I couldn't find a way.
Your suggestion sounds good, but I'd like to hear more.